python2/python/plan.sh

```diff
@@ -43,6 +40,13 @@ do_prepare() {
   sed -i -re "/(SSL=|_ssl|-DUSE_SSL|-lssl).*/ s|^#||" Modules/Setup.dist
 }
 
+do_setup_environment() {
+   set_buildtime_env PYTHON_SITE_PACKAGES "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
+   set_runtime_env PYTHON_SITE_PACKAGES "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
+   push_buildtime_env PYTHONPATH "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
+   push_runtime_env PYTHONPATH "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
```

Comment:
You can `$PYTHON_SITE_PACKAGES` as such
```bash
  push_buildtime_env PYTHONPATH "${pkg_prefix}${PYTHON_SITE_PACKAGES}"
  push_runtime_env PYTHONPATH "${pkg_prefix}/${PYTHON_SITE_PACKAGES}"
```

Reply:
hm, appears that ${PYTHON_SITE_PACKAGES} wasn't a defined variable when doing that, so going to revert back to the un-dry method.

also had a thought - since the `lib/python2.7/site-packages` directory is empty and immutable, is there any reason to add it to the PYTHONPATH at all?   should the `lib/python2.7` directory above be in the PYTHONPATH, or does python automagically include it?

Reply:
I think I just answered my own questions by reading some docs ( https://docs.python.org/2/using/cmdline.html#environment-variables ) -   it seems the python hab package doesn't need to set PYTHONPATH at all.